data_structures/disjoint_set/disjoint_set_2.py

```diff
@@ -0,0 +1,91 @@
+n = 9
```

Comment:
No docs is provided for this PR, you should look into other merged PRs for docs syntax.